projects/netgrif-components-core/src/lib/actions/model/front-action-definitions.ts

```diff
@@ -0,0 +1,28 @@
+import {FrontActionDefinition} from "./front-action-definition";
+import {Injector} from "@angular/core";
+import {Router} from "@angular/router";
+import {FrontAction} from "../../data-fields/models/changed-fields";
+import {TaskContentService} from "../../task-content/services/task-content.service";
+import {TaskDataService} from "../../task/services/task-data.service";
+import {AfterAction} from "../../utility/call-chain/after-action";
+
+export const redirectAction: FrontActionDefinition = {
```

Comment:
Maybe it would be better to have one action per file or at least group only related actions in one file (for example dialog actions, router actions, etc.)

Reply:
Done